src/main/resources/templates/menubar.html

```diff
@@ -59,15 +59,14 @@
                         </button>
                     </form>
                 </li>
-                <li class="nav-item dropdown" sec:authorize="isAuthenticated() and hasAuthority('ROLE_OOTB')">
+                <li class="nav-item dropdown" sec:authorize="isAuthenticated()" th:if="${@realm.isOotb()}" ng-click="loadAvailableProfiles()">
```

Comment:
You should be able to load the `ootb.active.user.profiles.json` file directly in Angular.js without having to go through an API endpoint. This should simplify the code a lot.